modules/oadp-multiple-bsl-use-case.adoc

```diff
@@ -0,0 +1,177 @@
+----
+<1> Specify the name of the cloud credentials file for AWS S3.
+
+. Create the second `Secret` for MCG with a custom name by running the following command.
++
+[source, terminal]
+----
+oc create secret generic mcg-secret -n openshift-adp --from-file cloud=<MCG_credentials_file_name> # <1>
+----
+<1> Specify the name of the cloud credentials file for MCG. Note the name of the custom secret `mcg-secret`.
+
+. Configure the DPA with the two BSLs as shown in the following example.
++
+[source, yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: two-bsl-dpa
+  namespace: openshift-adp  
+spec:
+  backupLocations:
+  - name: aws
+    velero:
+      config:
+        profile: default
+        region: <region> # <1>
+      credential:
+        key: cloud
+        name: cloud-credentials
+      default: true
+      objectStorage:
+        bucket: <bucket_name> # <2>
+        prefix: velero
+      provider: aws
+  - name: mcg
+    velero:
+      config:
+        insecureSkipTLSVerify: "true"
+        profile: noobaa
+        region: noobaa
+        s3ForcePathStyle: "true"
+        s3Url: <s3_url> # <3>
+      credential:
+        key: cloud
+        name: mcg-secret # <4>
+      objectStorage:
+        bucket: <bucket_name_mcg> # <5>
+        prefix: velero
+      provider: aws
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+    velero:
+      defaultPlugins:
+      - openshift
+      - aws
+----
+<1> Specify the AWS region for the bucket.
+<2> Specify the AWS S3 bucket name.
+<3> Specify the URL of the S3 endpoint for MCG.
+<4> Specify the name of the custom secret for MCG storage.
+<5> Specify the MCG bucket name.
+
+. Create the DPA by running the following command.
++
+[source, terminal]
+----
+oc create -f <dpa_file_name> # <1>
+----
+<1> Specify the file name of the DPA you configured in an earlier step.
+
+. Verify if the DPA has reconciled by running the following command.
++
+[source, terminal]
+----
+oc get dpa -o yaml
+----
+
+. Verify if the BSL status is available by running the following command.
++
+[source, terminal]
+----
+oc get bsl
+----
+
+. Create a backup `CR` with the default BSL. In the following example note that, the `storageLocation` field is not specified in the backup `CR`. 
++
+[source, yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: test-backup1
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - <mysql_namespace> # <1>
+  defaultVolumesToFsBackup: true
+----
+<1> Specify the namespace for the `MySQL` application installed in the cluster.
+
+. Create a backup by running the following command.
++
+[source, terminal]
+----
+oc apply -f <backup_file_name> # <1>
+----
+<1> Specify the name of the backup `CR` file.
+
+. Verify that the backup completed with the default BSL by running the following command.
++
+[source, terminal]
+----
+oc get backup <backup_name> -o yaml # <1>
```

Comment:
@weshayutin how this is done in other parts of docs? do we prefer to tell users to run `velero` command?

Reply:
We have both well documented and either is fine imho.
* https://docs.openshift.com/container-platform/4.16/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.html
* https://docs.openshift.com/container-platform/4.16/backup_and_restore/application_backup_and_restore/troubleshooting.html#migration-debugging-velero-resources_oadp-troubleshooting